src/lib/notion.ts

```diff
@@ -1,11 +1,23 @@
 /* eslint-disable @typescript-eslint/naming-convention */
 import { Client } from '@notionhq/client';
 
-const notion = new Client({
-  auth: process.env.NOTION_TOKEN,
-});
+const isNotionEnabled = () => {
+  return !!process.env.NOTION_TOKEN;
+};
+
+let notion: Client;
+
+if (isNotionEnabled) {
```

Comment:
Since `isNotionEnabled` is a function, shouldn't this be `if (isNotionEnabled()) {` instead of `if (isNotionEnabled) {`? What is even better is to keep it as it is and change `isNotionEnabled` from being a function to just a fixed const e.g. `const isNotionEnabled = !!process.env.NOTION_TOKEN`.